### DISCUSSION TITLE
Costume Editor Shortcuts

### DISCUSSION THREAD

**Discussion by HighFlyingDonkey** [ScratchAddons/ScratchAddons#5935] [Ideas]
Hello fellow scratchers!

I have got an idea for Scratch Addons, shortcuts in the custome editor.

I feel like every art program has this, but scratch vector/bitmap not. As you probably know, shortcuts enable you to work much faster.  It would be useful to have shortcuts for the following actions:

- Brush
- Erasing
- Fill
- _Color Picking_
- Circle
- Rectangle
- Line
- Reshape
- Etc.

Thanks for reading,

A random scratcher

**Comment by Platformer444** [comment]
There are shortcuts already available in vanilla scratch iirc

**Reply by HighFlyingDonkey** [reply]
Could I ask what that is?

**Reply by WorldLanguages** [reply]
I don't think vanilla Scratch has shortcuts to switch tools (see https://github.com/scratchfoundation/scratch-paint/issues/990)
But there are shortcuts for cut, group, ungroup, and some others already

**Reply by Samq64** [reply]
I thought there were too, but it looks like it's a TurboWarp thing.

**Reply by CST1229** [reply]
It _is_ a TurboWarp thing; see https://github.com/TurboWarp/scratch-paint/commit/e4624a76365d3df3bbedab5c9ac19c5bcd9c177a.

**Comment by Secret-chest** [comment]
Please make them use modifiers
